src/engines/html5/html5.js

```diff
@@ -265,23 +265,16 @@ export default class Html5 extends FakeEventTarget implements IEngine {
         Utils.Dom.removeAttribute(this._el, 'src');
       }
       this._init(this._source, this._config);
-      this._eventManager.listenOnce(this, Html5EventType.LOADED_METADATA, () => {
-        if (this._lastTimeDetach) {
-          this.currentTime = this._lastTimeDetach;
-          this._lastTimeDetach = null;
-        }
-      });
     }
   }
-
   /**
    * Listen to the video element events and triggers them from the engine.
    * @public
    * @returns {void}
    */
   detachMediaSource(): void {
     if (this._mediaSourceAdapter) {
-      this._lastTimeDetach = this.currentTime;
+      this._config.playback.startTime = this.currentTime;
```

Comment:
I think this will cause player to start at this time also on change media, which is not desired or expected by app

Reply:
Right, I'll reset it on 'attachMediaSource'

Reply:
I think playback startTime will trigger the hlsjs startTime which is buggy...

Reply:
It'll but I checked on hls.js issues and I didn't find an open issue about that.(there was an issue that closed)

Reply:
What should be buggy there?

Reply:
We already have a use case where we do this, when coming back from casting on chromecast back to browser, and we saw it tends to fail.
Need to verify this VERY good before we come back to this solution.

Reply:
I changed it back for seeking, it failed on LG